Source/NSWindow.m

```diff
@@ -2702,6 +2702,8 @@ - (NSScreen *) screen
         }
     }
   ASSIGN(_screen, [self _screenForFrame: _frame]);
+  [nc postNotificationName: NSWindowDidChangeScreenNotification
```

Comment:
This did not work as you did not check whether the screen did change here.

Reply:
This code is outdated.